restart.html

```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE html>
+<html lang="ENGLISH">
+<head>
+    <meta charset="UTF-8">
+    <title> Becks Webpage</title>
+    <link rel="stylesheet" href="restart.css" type="text/css"/>
+</head>
+<style> 
+h1,h2,h3 {text-align: center;}
+p {text-align: left;}
+</style>
+<body>
+
+<div id="main">
```

Comment:
Semantic HTML. It's better to use HTML elements that are made for each part of the site, rather than divs with different classes.

```suggestion
<main>
```